cymbaline/generators.py

```diff
@@ -0,0 +1,63 @@
+from cymbaline.data import DataProvider
+from cymbaline.language import LanguageToolkit
+
+import random
+
+
+class Generator:
+    def __init__(self, context_word=None, context_vector=None):
+    """Initiates Generator with either single word or vector, which defines the context"""
+        if(context_word == None and context_vector == None):
```

Comment:
In Python, we tend to stay away form unnecessary parenthesis.
```suggestion
        if context_word == None and context_vector == None:
```

Reply:
This is not C programming language indeed.